source/material_model/melt_simple.cc

```diff
@@ -0,0 +1,699 @@
+                             Patterns::Double (0),
+                             "The solidus temperature change for a depletion of 100\\%. For positive "
+                             "values, the solidus gets increased for a positive peridotite field "
+                             "(depletion) and lowered for a negative peridotite field (enrichment)."
+                             "Scaling with depletion is linear. Only active when fractional melting "
+                             "is used. "
+                             "Units: $K$.");
+          prm.declare_entry ("A1", "1085.7",
+                             Patterns::Double (),
+                             "Constant parameter in the quadratic "
+                             "function that approximates the solidus "
+                             "of peridotite. "
+                             "Units: $°C$.");
+          prm.declare_entry ("A2", "1.329e-7",
+                             Patterns::Double (),
+                             "Prefactor of the linear pressure term "
+                             "in the quadratic function that approximates "
+                             "the solidus of peridotite. "
+                             "Units: $°C/Pa$.");
+          prm.declare_entry ("A3", "-5.1e-18",
+                             Patterns::Double (),
+                             "Prefactor of the quadratic pressure term "
+                             "in the quadratic function that approximates "
+                             "the solidus of peridotite. "
+                             "Units: $°C/(Pa^2)$.");
+          prm.declare_entry ("B1", "1475.0",
+                             Patterns::Double (),
+                             "Constant parameter in the quadratic "
+                             "function that approximates the lherzolite "
+                             "liquidus used for calculating the fraction "
+                             "of peridotite-derived melt. "
+                             "Units: $°C$.");
+          prm.declare_entry ("B2", "8.0e-8",
+                             Patterns::Double (),
+                             "Prefactor of the linear pressure term "
+                             "in the quadratic function that approximates "
+                             "the  lherzolite liquidus used for "
+                             "calculating the fraction of peridotite-"
+                             "derived melt. "
+                             "Units: $°C/Pa$.");
+          prm.declare_entry ("B3", "-3.2e-18",
+                             Patterns::Double (),
+                             "Prefactor of the quadratic pressure term "
+                             "in the quadratic function that approximates "
+                             "the  lherzolite liquidus used for "
+                             "calculating the fraction of peridotite-"
+                             "derived melt. "
+                             "Units: $°C/(Pa^2)$.");
```

Comment:
Not sure how this is going to show up in latex. You may have to write `\\circ` or `{}^\circ`.